packages/frontend/src/components/admin/tabs/EventsTab.tsx

```
@@ -0,0 +1,139 @@
+
+  const columns = [
+    { accessorKey: 'id', header: 'ID' },
+    { accessorKey: 'title', header: 'Titre' },
+    { accessorKey: 'date', header: 'Date' },
+    { accessorKey: 'lieu', header: 'Lieu' },
+    { accessorKey: 'price', header: 'Prix' },
+    { accessorKey: 'status', header: 'Statut' },
+    {
+      id: 'actions',
+      header: 'Actions',
+      cell: ({ row }: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)